os/include/tinyara/testcase_drv.h

```diff
@@ -74,6 +74,7 @@
 #define TESTIOC_GROUP_REMOVECHILDREN_TEST      _TESTIOC(17)
 #endif
 #define TESTIOC_TASK_INIT_TEST                 _TESTIOC(18)
+#define TESTIOC_COMPRESSION_TEST	       _TESTIOC(19)
```

Comment:
Sorry for another comments.
1. Could you align `_TESTIOC(10)`?
2. Could you leave the result logs in commit description?

Reply:
Changes made. Added the result logs in commit description.